tests/integration/security/cert_mtls/main_test.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2020 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package certmtls
+
+import (
+	"testing"
+
+	"istio.io/istio/tests/integration/security/util/cert"
+
+	"istio.io/istio/pkg/test/framework"
+	"istio.io/istio/pkg/test/framework/components/galley"
+	"istio.io/istio/pkg/test/framework/components/istio"
+	"istio.io/istio/pkg/test/framework/components/pilot"
+	"istio.io/istio/pkg/test/framework/label"
+	"istio.io/istio/pkg/test/framework/resource"
+	"istio.io/istio/pkg/test/framework/resource/environment"
+)
+
+var (
+	inst istio.Instance
+	g    galley.Instance
+	p    pilot.Instance
+)
+
+func TestMain(m *testing.M) {
+	// This test verifies that the certificate issued by CA to the sidecar
+	// is as expected and that strict mTLS works as expected.
+	framework.
+		NewSuite("cert_mtls_test", m).
+		// k8s is required because the plugin CA key and certificate are stored in a k8s secret.
+		RequireEnvironment(environment.Kube).
+		RequireSingleCluster().
+		Label(label.CustomSetup).
+		SetupOnEnv(environment.Kube, istio.Setup(&inst, nil, cert.CreateCASecret)).
```

Comment:
how is this any different than the plugin_ca_cert? Why does it need to be its own suite? It looks like the setup is the same

Reply:
Done: merged plugin_ca_cert into TestCertMtls.